staging/src/k8s.io/apiserver/pkg/util/flowcontrol/gen_test.go

```diff
@@ -545,10 +547,12 @@ func genResourceRule(rng *rand.Rand, pfx string, mayMatchClusterScope, matchAllR
 			[]string{pfx + "-v4", pfx + "-v5", pfx + "-v6"},
 			[]string{pfx + ".g4", pfx + ".g5", pfx + ".g6"},
 			[]string{pfx + "-r4s", pfx + "-r5s", pfx + "-r6s"},
-			skipNSs)
+			skipNSs,
+			[]string{pfx + "-nn4", pfx + "-nn5", pfx + "-nn6"},
+		)
 	}
 	// choose a proper subset of fields to wildcard; only matters if not matching all
-	starMask := rng.Intn(15)
+	starMask := rng.Intn(31)
```

Comment:
Shouldn't this be Intn(32)? That would generate between [0, 32), so 31 will also be possible (all five set to All/Every)

https://golang.org/pkg/math/rand/#Rand.Int31n

Reply:
Yes, but I think we should save this change for a followup. Looks like existing constant should be 16.

Consequence doesn't seem to be very bad, just can't have all on at once. Actually the comment isn't clear but it might be trying to say that it's deliberate. @MikeSpreitzer do you recall if this is deliberate or a bug?

Reply:
i think we should move this conversation to https://github.com/kubernetes/kubernetes/pull/94244?